src/viam/ur/module/ur_arm.cpp

```diff
@@ -399,11 +399,11 @@ void URArm::move_to_joint_positions(const std::vector<double>& positions, const
     write_waypoints_to_csv(filename, waypoints);
 
     // move will throw if an error occurs
-    move_joint_space_(std::move(rlock), std::move(waypoints), unix_time);
+    move_joint_space_(std::move(rlock), std::move(waypoints), MoveOptions{}, unix_time);
```

Comment:
A little surprising to me that `move_through` takes options but `move_to` doesn't?